exec/vm.go

```diff
@@ -84,6 +91,7 @@ func NewVM(module *wasm.Module) (*VM, error) {
 	vm.globals = make([]uint64, len(module.GlobalIndexSpace))
 	vm.newFuncTable()
 	vm.module = module
+	vm.RecoverExecutionPanic = false
```

Comment:
There isn't a need to do this, since `false` is already the zero value for `bool` type.

Reply:
I'm aware. I always do this so that the code is more understandable to people who get started with a complex codebase. This way, a simple `grep -r RecoverExecutionPanic` will reveal where it is initialized.